addons/project_sale_expense/tests/test_project_profitability.py

```diff
@@ -82,18 +83,21 @@ def test_project_profitability(self):
         sequence_per_invoice_type = project._get_profitability_sequence_per_invoice_type()
         self.assertIn('expenses', sequence_per_invoice_type)
         expense_sequence = sequence_per_invoice_type['expenses']
-        billed = -expense.untaxed_amount_currency - expense_foreign.untaxed_amount_currency * 0.2  # 280 + 350 * 0.2 = 350
+        billed_expense_sheet = -expense.untaxed_amount_currency
+        billed_foreign_sheet = -expense_foreign.untaxed_amount_currency * 0.2
+        billed = billed_expense_sheet + billed_foreign_sheet  # -280.0 - 175.0 * 0.2 = -315.0
```

Comment:
As I reuse the `billed_expense_sheet` variable further in the code, I thought it was a good idea to keep it

Reply:
for only one more occurrence, i dont think its necessary.
but if you really want to keep it, maybe either get rid of the 'billed_foreign_sheet', since this one is not reused, or set both in one line like this :
```py
billed_expense_sheet, billed_foreign_sheet = -expense.untaxed_amount_currency, -expense_foreign.untaxed_amount_currency * 0.2
```